rfc/003-account-hierarchy.md

```diff
@@ -0,0 +1,37 @@
+# RFC for openMFP Account Hierarchy
+
+## Context and Problem Statement
+
+This document provides an outline of how and what should be achieved with an account hierarchy model within openMFP. An account will have the ability to benefit from other accounts that it is a child of, e.g., by inheriting extensions provided by its parents. This will allow for easier distribution of central extensions and give more structure to the entire account setup.
+
+## Decision Drivers
+
+- Accounts within the same hierarchy must be able to inherit extensions from their parents until the root is reached.
+- There is (theoretically) no limit on the depth of the hierarchy.
+- There will be a root account (root of the hierarchy tree) that provides the necessary foundation for all other accounts.
+- "First-Level Accounts" have a special role and act as a top-level separator for configurations.
+
+## Options Considered
```

Comment:
In an RFC - given that you only outline one Option, I'd skip this part as its not a ADR. Rather outline the proposal